code/modules/mob/living/carbon/metroid/items.dm

```diff
@@ -171,7 +171,7 @@
 		if(M.mind)
 			user << SPAN_WARNING(" The slime resists!")
 			return ..()
-		var/mob/living/simple_animal/adultslime/pet = new /mob/living/simple_animal/adultslime(M.loc)
```

Comment:
what does crafting have to do with slimes?

Reply:
Nothing